### PR TITLE
Add siteNav bar

### LINKED ISSUE #57: Add a site nav bar
Can be similar to this:
image

Screenshot from https://nus-cs2103.github.io/website-base/se-book-adapted/

### PR DESCRIPTION
**Pull request by Tejas2805**
Fix #57

### CODE REVIEW

**Comment by Tejas2805** [comment]
@damithc ready for review

**Comment by Tejas2805** [comment]
> Looks good, but there are some redundant changes in the diff though.

There was no EOF line in those files so I added them. Want me to remove them?

File: _markbind/navigation/mainNav.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+
+<span id="toc">
```
Comment (damithc) [inline _markbind/navigation/mainNav.md]: Do we need this?

**Comment by damithc** [comment]
> There was no EOF line in those files so I added them. Want me to remove them?

In that case, OK to keep.

File: _markbind/navigation/mainNav.md (hunk 1 above)
Comment (Tejas2805) [reply _markbind/navigation/mainNav.md]: Optional. So, have removed it.

**Comment by Tejas2805** [comment]
@damithc Any other update required?

**Comment by damithc** [comment]
Thanks for the contribution @Tejas2805

**Comment by Tejas2805** [comment]
Do let me know if there is any other issue that I can help with.

**Pull request merged**